TestGlue/MobileCoinMinimalObjcTest.h

```diff
@@ -0,0 +1,14 @@
+//
+//  Copyright (c) 2021 Open Whisper Systems. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MobileCoinProtosObjcTest : XCTestCase
```

Comment:
I ended up using tests in the `SignalUI` target, so that we can compare the output of this minimal fork and the real MobileCoin SDK.